### ISSUE TITLE
adding a new meal crashed the site

### ISSUE THREAD

**Issue by mattlub** [facn2/The-Munching-VASS#56]
on heroku, with what I would assume would be valid data.

**Comment by marisid** [comment]
Yep, I cannot add a new meal either. Check out your error handling - heroku timed out after a while.
